### PR TITLE
[ECP-8448]Indirectly test private methods of AuthorisationWebhookHandler

### PR DESCRIPTION
**Pull request by khushboo-singhvi**
**Description**
Unit test for public and private methods of AuthorisationWebhookHandler

### CODE REVIEW

File: Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php [hunk 1]
```diff
@@ -1,130 +1,363 @@
...
+        $webhookHandler = $this->createAuthorisationWebhookHandler(
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
```
Comment (candemiralp) [inline Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php:241]: You can skip sending values for optional arguments. They are already `null` by default.

File: Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php [hunk 2]
```diff
@@ -1,130 +1,363 @@
...
+        $webhookHandler = $this->createAuthorisationWebhookHandler(
+            null,
+            $this->orderHelperMock,
+            $this->caseManagementMock,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
```
Comment (candemiralp) [inline Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php:281]: You can skip sending values for optional arguments. They are already null by default.

File: Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php [hunk 3]
```diff
@@ -1,130 +1,363 @@
...
+        $webhookHandler = $this->createAuthorisationWebhookHandler(
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
```
Comment (candemiralp) [inline Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php:333]: You can skip sending values for optional arguments. They are already null by default.

File: Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php [hunk 4]
```diff
@@ -1,130 +1,363 @@
...
```
Comment (candemiralp) [inline Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php:205]: You can skip sending values for optional arguments. They are already null by default.

**Review by candemiralp — CHANGES_REQUESTED** [review]

**Review by hossam-adyen — APPROVED** [review]

**Review by candemiralp — APPROVED** [review]

**Pull request merged**